tests/data/model_params.json

```diff
@@ -1,160 +1,128 @@
-                        "min": -0.0095,
-                        "num_value": 651,
-                        "scale": 0.0039
-                    },
-                    "sequential": null
-                }
-            },
-            "terminal_diameters": {
-                "apical": {
-                    "distribution": "exponnorm",
-                    "params": {
-                        "a": 2.891,
-                        "loc": 0.2531,
-                        "max": 0.9834,
-                        "min": 0.22,
-                        "num_value": 4525,
-                        "scale": 0.0927
-                    },
-                    "sequential": null
-                },
-                "basal": {
-                    "distribution": "exponnorm",
-                    "params": {
-                        "a": 0.9409,
-                        "loc": 0.4201,
-                        "max": 1.05,
-                        "min": 0.2308,
-                        "num_value": 2717,
-                        "scale": 0.1976
-                    },
-                    "sequential": null
-                }
-            },
-            "trunk_diameters": {
-                "apical": {
-                    "distribution": "exponnorm",
-                    "params": {
-                        "a": 1.3373,
-                        "loc": 3.1955,
-                        "max": 8.8355,
-                        "min": 1.7995,
-                        "num_value": 72,
-                        "scale": 1.2579
-                    },
-                    "sequential": null
-                },
-                "basal": {
-                    "distribution": "exponnorm",
-                    "params": {
-                        "a": 1.4124,
-                        "loc": 1.0764,
-                        "max": 3.6015,
-                        "min": 0.52,
-                        "num_value": 498,
-                        "scale": 0.5655
-                    },
-                    "sequential": null
-                }
-            },
-            "trunk_tapers": {
-                "apical": {
-                    "distribution": "skewnorm",
-                    "params": {
-                        "a": 0.7162,
-                        "loc": 0.0299,
-                        "max": 0.0834,
-                        "min": 0.0004,
-                        "num_value": 44,
-                        "scale": 0.0275
-                    },
-                    "sequential": null
-                },
-                "basal": {
-                    "distribution": "skewnorm",
-                    "params": {
-                        "a": 4,
-                        "loc": -0.0031,
-                        "max": 0.0637,
-                        "min": 0.0,
-                        "num_value": 294,
-                        "scale": 0.0285
-                    },
-                    "sequential": null
-                }
+    "sibling_ratios": {
```

Comment:
Why did you change these values? Changing them also changes expected values in tests, so it's harder to know if the new expected values changed because of changes in inputs or in the code.

Reply:
these are the expected values to test against, no? There was a bug in t he test, you didn't add `assert` to compare things, so half of the test where outdated, so these large changes

Reply:
Oh boy :facepalm: 
Ok, I understand why you changed :-)